### PR TITLE
Fix EmptySpotDesignator (https://github.com/tue-robotics/tue_robocup/issues/185 )

### PR DESCRIPTION
**Pull request by LoyVanBeek**

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
@LoyVanBeek Ready to be tested?

**Comment by LoyVanBeek** [comment]
This has been tested on Amigo last week and @jlunenburg said he would merge it today.

**Comment by jlunenburg** [comment]
Didn't get to this because we needed to fix some other stuff for the R5-Cop demo

**Comment by LoyVanBeek** [comment]
This works on the real robot. 
screenshot from 2017-04-04 19-20-28

File: robot_smach_states/src/robot_smach_states/util/designators/ed_designators.py [hunk 1]
```diff
@@ -210,11 +213,11 @@ def _resolve(self):
         rospy.logerr("No entities found in {0}".format(self))
         return None
 
-    # def __repr__(self):
-    #     criteria_code = [inspect.getsource(criterium).strip().replace('\\n', '\n') for criterium in self.criteriafuncs]
+        # def __repr__(self):
```
Comment (jlunenburg) [inline robot_smach_states/src/robot_smach_states/util/designators/ed_designators.py]: Deprecated?

File: robot_smach_states/src/robot_smach_states/util/designators/ed_designators.py [hunk 2]
```diff
@@ -101,19 +98,19 @@ def _resolve(self):
         rospy.logerr("No entities found in {0}".format(self))
         return None
 
-    # def __repr__(self):
-    #     return "EdEntityCollectionDesignator(robot, type={0}, center_point={1}, radius={2}, id={3}, parse={4}, criteriafuncs={5})".format(
-    #         self.type, str(self.center_point).replace("\n", " "), self.radius, self.id, self.parse, self.criteriafuncs)
+        # def __repr__(self):
```
Comment (jlunenburg) [inline robot_smach_states/src/robot_smach_states/util/designators/ed_designators.py]: Deprecated?

**Pull request merged**